github/orgs_projects.go

```diff
@@ -12,7 +12,7 @@ import (
 
 // ListProjectsOptions specifies the optional parameters to the
 // OrganizationsService.ListProjects method.
-type ListProjectsOptions struct {
+type ProjectsListOptions struct {
```

Comment:
yeah, missed the comment, my fail :)
Are we ok leaving the definition in `orgs_projects.go`?